app/assets/stylesheets/custom.css.scss

```diff
@@ -26,11 +26,63 @@ body {
   padding: 5px;
 }
 
-.hero-unit {
+.hero-unit1 {
   background-image: image-url("volunteer-couple-hero.png");
-  background-size:cover;
-  height:600px;
+  background-size: cover;
+  filter: gray;
+  height: 650px;
+}
+
+.hero-text {
+  color: $site-white;
+  padding-top: 450px;
+}
+
+.notice {
+  color: $site-orange;
+}
+
+.sub-notice {
+  color: $site-light-grey;
+}
+
+.btn-landing {
+  color: $site-white;
+  background-color: $site-orange;
+}
+
+.point {
+  border: $site-brown 1px solid;
+  padding: 10px 30px 20px 30px;
+  margin: 0px 20px 0px 20px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0px 20px` instead of `0px 20px 0px 20px`<br>`0px` should be written without units as `0`